gst/nnstreamer/elements/gsttensor_transform.c

```diff
@@ -238,6 +245,11 @@ gst_tensor_transform_class_init (GstTensorTransformClass * klass)
       g_param_spec_string ("apply", "Apply", "Select tensors to apply, "
           "separated with ',' in case of multiple tensors. Default to apply all tensors.",
           "", G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+  g_object_class_install_property (gobject_class, PROP_TRANSPOSE_RANK,
+      g_param_spec_uint ("transposelimit", "Transpose limit",
+          "The rank of transpose limit",
```

Comment:
"The rank limit of transpose, which varies per version of nnstreamer and may be lower than the global rank limit if it is over 4."

Reply:
updated!
Thank you :)